### PR TITLE
Muflus autoattack

### LINKED ISSUE #612: [FEATURE] Return skill orientation
We should add the skill orientation on every skill message so we could change the orientation on every client.

### PR DESCRIPTION
**Pull request by manucamejo**
- Closes #612 
- Muflus autoattack
- Backend returns player direction (movement, basic attack) #221

### CODE REVIEW

File: server/native/gamestate/src/game.rs [hunk 1]
```diff
@@ -434,6 +460,9 @@ impl GameState {
             );
             projectiles.push(projectile);
             (*next_projectile_id) += 1;
+
+            //let attacking_player = GameState::get_player_mut(players, attacking_player.id)?;
```
Comment (ncontinanza) [inline server/native/gamestate/src/game.rs:464]: Idem here

File: server/native/gamestate/src/game.rs [hunk 2]
```diff
@@ -462,104 +491,57 @@ impl GameState {
...
         direction: &RelativePosition,
+        attack_range: f64,
     ) -> Result<Vec<u64>, String> {
         let attack_dmg = attacking_player.basic_skill_damage() as i64;
-        let attack_position = Position::new(
-            (attacking_player.position.x as i64 - (direction.y * 200.) as i64) as usize,
-            (attacking_player.position.y as i64 + (direction.x * 200.) as i64) as usize,
-        );
-
         // TODO: This should be a config of the attack
```
Comment (ncontinanza) [inline server/native/gamestate/src/game.rs:501]: I think this TODO is now outdated and could be removed

File: server/native/gamestate/src/game.rs [hunk 3]
```diff
@@ -462,104 +491,57 @@ impl GameState {
...
-            );
-            if matches!(attacked_player.status, Status::DEAD) {
-                kill_count += 1;
+                (vec![player_id], direction)
             }
-        }
-        GameState::attack_mirrored_player(uma_mirroring_affected_players, players)?;
-        add_kills(players, attacking_player.id, kill_count).expect("Player not found");
+            None => {
+                //attacking_player.direction = *direction;
```
Comment (ncontinanza) [inline server/native/gamestate/src/game.rs:536]: Same as before

**Review by ncontinanza — DISMISSED** [review]
Nice job! Just a few questions about some comments left in the code, but we're good to go

**Review by ncontinanza — APPROVED** [review]

File: server/lib/dark_worlds_server/engine/runner.ex [hunk 4]
```diff
@@ -82,6 +82,7 @@ defmodule DarkWorldsServer.Engine.Runner do
     Process.send_after(self(), :all_characters_set?, @character_selection_check_ms)
     Process.send_after(self(), :character_selection_time_out, @character_selection_timeout_ms)
     Process.send_after(self(), :check_player_amount, @player_check)
+    Process.send_after(self(), :shrink_map, @map_shrink_wait_ms)
```
Comment (tvillegas98) [inline server/lib/dark_worlds_server/engine/runner.ex:85]: Remove this

**Review by ncontinanza — APPROVED** [review]

**Review by tvillegas98 — APPROVED** [review]

**Pull request merged**